src/Pickles/Pickles.Test/BaseFixture.cs

```diff
@@ -43,9 +43,12 @@ protected IContainer Container
                 if (this.container == null)
                 {
                     var builder = new ContainerBuilder();
+
+                    var configuration = new Configuration() { ExcludeTags = "ignore-tag" };
```

Comment:
I'd prefer it if you also renamed the ignore-tag in the test data to exclude-tag - here and in the other test cases